acdc-ws/app/tasks/DataInstExpirationTask.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+package tasks
+
+import java.net.InetAddress
+import java.time.LocalDateTime
+import java.util.UUID
+import javax.inject.Inject
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration._
+
+import akka.actor.ActorSystem
+import play.api.Logging
+import services.DatabaseService
+
+import com.salesforce.mce.acdc.db.{DatasetInstanceQuery, ExpirationTaskQuery}
+import utils.DbConfig
+
+class DataInstExpirationTask @Inject()(
+  actorSystem: ActorSystem,
+  dbService: DatabaseService,
+  dbConfig: DbConfig
+)(implicit
+  ec: ExecutionContext
+) extends Logging {
+
+  val refreshDelay: FiniteDuration = 12.hour
+  val hostName = (InetAddress.getLocalHost.getHostName + UUID.randomUUID().toString).take(256)
```

Comment:
let's start with the `randomUUID` and also have a separator between them to make sure we are not truncating that.

Reply:
so truncating hostname instead?